commcare-cloud/README.md

```diff
@@ -0,0 +1,84 @@
+# Set up commcarehq-ansible
+
+Clone the commcarehq-ansible repo (suggested location alongside commcare-hq):
+
+```
+git clone https://github.com/dimagi/commcarehq-ansible.git
+cd commcarehq-ansible
+```
+
+Now make a virtualenv for ansible:
+
+```
+mkvirtualenv ansible
+```
+
+If you want `workon ansible` to always bring you to this directory, then you can also run
+
+```
+pwd > ~/.virtualenvs/ansible/.project
+```
+
+at this time.
+
+**Note**: If you already have commcarehq-ansible cloned, then just enter that directory
+and update it with
+
+```
+git pull
+```
+
+# Install commcare-cloud
+
+```
+workon ansible
+```
+
+```
+pip install pip --upgrade
+```
+
+```
+pip install -e commcare-cloud
+pip install -r ansible/requirements.txt  # to be able to run ansible commands
+ansible-galaxy install -r ansible/requirements.yml  # to be able to run ansible commands
+pip install -r fab/requirements.txt  # to be able to run fab commands
+```
```

Comment:
How would you feel about a top level `requirements.txt` file like:
```
-e commcare-cloud
-r ansible/requirements.txt
-r fab/requirements.txt
```
That way everything is updated in one go

Reply:
Thanks for bringing this up. I was initially going to tell you that I actually had it this way in my first commit on this PR, but that it complained because both `ansible/requirements.txt` and `fab/requirements.txt` contained ansible (the same version even). But then I thought I'd give it one more shot, and was able to ascertain that that only happens with an old version of pip. So as long as you `pip install pip --upgrade` (which I later added to this README and which you have to do anyway for some of the installs), you can actually put these in a single file as I tried and as you just suggested!

I'm going to go ahead and do that. Thanks!